monero-node/docker-compose.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      # <app-id>_<web-container-name>_1
+      # e.g. 'btc-rpc-explorer_web_1'
+      # Note that the '_1' at the end is needed
+      APP_HOST: monero-node_monerod_1
+      APP_PORT: 18089
+      PROXY_AUTH_ADD: "false"
+  
+  monerod:
+    image: sethsimmons/simple-monerod:v0.18.2.2
+    command: "--rpc-restricted-bind-ip=0.0.0.0 --rpc-restricted-bind-port=18089 --public-node --no-igd --no-zmq --enable-dns-blocklist"
+    restart: unless-stopped
+    stop_grace_period: 15m30s
+    volumes:
+      - "${APP_MONERO_DATA_DIR}:/home/monero/.bitmonero"
+    ports:
+      - "${APP_MONERO_P2P_PORT}:${APP_MONERO_P2P_PORT}"
+      # - "${APP_MONERO_RPC_PORT}:${APP_MONERO_RPC_PORT}"
+    networks:
+      default:
+        ipv4_address: $APP_MONERO_NODE_IP
+  # If your app has more services, like a database container, you can define those
```

Comment:
Likely can remove these lines?

Reply:
I'm not sure, but as I understand it, the IP needs to be set to "something"; in the app submission guide, Umbrel team explains that this will be set to some unique value during their review.

Seems redundant for me, too, but wanted to stick to the script :)

Reply:
Oops, sorry, this selected the wrong lines!

I just meant the commented out template lines starting with 26.

Reply:
removed the comments, thanks!